tools/ddrgen/src/scanners/dwarf/AixDwarfParser.hpp

```diff
@@ -19,6 +19,19 @@
 #include "config.hpp"
 #include "DwarfFunctions.hpp"
 
+/* PTR_SIZE varies for 32bit, 64bit */
+const int PTR_SIZE = 8;
+
+const string BUILT_IN_TYPES [36] = {"", "int", "char", "short", "long", "unsigned char",
+"signed char", "unsigned short", "unsigned int", "unsigned", "unsigned long", "void", 
+"float", "double", "long double", "integer", "boolean", "short real", "real", "stringptr",
+"character", "logical*1", "logical*2", "logical*4", "logical", "complex", "complex", "integer*1",
+"integer*2", "integer*4", "wchar", "long long", "unsigned long long", "logical*8", "integer*8",
```

Comment:
Well, I wouldn't deem some as not needed and delete them. I suggest creating one signed/unsigned for each possible size, then creating typedefs that reference them for the types of redundant sizes. That link suggests some are Fortran-specific.